### PR TITLE
[3.11] GH-98363: Update batched() recipe in docs to match 3.12

### PR DESCRIPTION
**Pull request by rhettinger**
* Issue: gh-98363

### CODE REVIEW

File: Doc/library/itertools.rst [hunk 1]
```diff
@@ -1248,25 +1248,25 @@ which incur interpreter overhead.
     [('a', 'b', 'c'), ('d', 'e', 'f')]
 
     >>> list(batched('ABCDEFG', 3))
-    [['A', 'B', 'C'], ['D', 'E', 'F'], ['G']]
+    [('A', 'B', 'C'), ('D', 'E', 'F'), ('G',)]]
```
Comment (hauntsaninja) [inline Doc/library/itertools.rst]:
```suggestion
    [('A', 'B', 'C'), ('D', 'E', 'F'), ('G',)]
```

**Review by CAM-Gerlach — APPROVED** [review]
LGTM from a docs perspective, FWIW. Should this be backported to 3.10 as well?

File: Doc/library/itertools.rst (hunk 1 above)
Comment (CAM-Gerlach) [reply Doc/library/itertools.rst]: Looks like this was fixed in af8ad77

**Comment by CAM-Gerlach** [comment]
Also, I noticed that though this PR was made from your fork per the usual flow. there is an apparent duplicate https://github.com/python/cpython/tree/batched_recipe_tuple branch on the upstream `cpython` repo. Does that need to be there, or can it be removed? It's the final blocker to implementing the final step of python/core-workflow#460 on `cpython`. Thanks!

**Comment by rhettinger** [comment]
@CAM-Gerlach Yes, go ahead and remove the other branch.

**Comment by CAM-Gerlach** [comment]
Thanks, removed! :+1:

**Pull request merged**